react/components/ProductDescription/index.js

```diff
@@ -1,35 +1,88 @@
-import React, { PureComponent } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl'
 
 import VTEXClasses from './CustomClasses'
 
+import './global.css'
+
 /**
  * Product Description Component.
- * Render the description of a product.
+ * Render the description of a product and it's technical specifications.
  */
-class ProductDescription extends PureComponent {
+class ProductDescription extends Component {
   render() {
+    const { specifications, skuName } = this.props
+
     return (
       <div className={`${VTEXClasses.PRODUCT_DESCRIPTION} ma2`}>
         <div className="f4 b ttu mb3">
           <FormattedMessage id="product-description.title" />
         </div>
+
         {this.props.children}
+
+        {specifications.length && (
```

Comment:
Não seria uma boa abstrair esse TechnicalSpecifications para outro componente? Igual tava antes. Tou achando o render muito extenso

Reply:
I agree.

Reply:
You mean putting in another file? Could be done, but I think its unnecessary because it won't be used anywhere else and the current file still small.